### ISSUE TITLE
Not able to compile libMR

### ISSUE THREAD

**Issue by urumican** [abhijitbendale/OSDN#2]
Hello,

I have some problems with the code.

When I use the command: 

`/compile.sh`

, I got:

```
gcc: error: unrecognized command line option ‘-fstack-protector-strong’
gcc: error: unrecognized command line option ‘-fstack-protector-strong’
error: command 'gcc' failed with exit status 1
cp: cannot stat ‘libmr.so’: No such file or directory
```

Do you now what kind gcc is needed for this?

**Comment by abhijitbendale** [comment]
The compilation process is using x86_64-linux-gnu-gcc for which you might need to install python-dev tools 

```bash
sudo apt-get install python-dev
```
If it doesnt work, let me know.

**Comment by urumican** [comment]
Thank you very much! Another thing is I have down load the validation image ground truth form ILSVRC2012 website. The name of the ground truth file is ILSVRC2012_validation_ground_truth.txt. I saw that the first element is 490. I think this means the image ILSVRC2012_val_00000001.JPEG (There is a snake in this image) belongs to category 490. But I find the corresponding category name in synset_words_caffe_ILSVRC12.txt is nothing about snake. 

Did I downloaded the wrong ground truth? Where Can I get the ground truth?

**Comment by abhijitbendale** [comment]
The synset_words_caffe_ILSVRC12.txt file corresponds to groundtruth validation file used for training the caffe model. Thus, when caffe using softmax predicts top class as 0 (i.e. the index), it corresponds to "n01440764 tench, Tinca tinca". This is part of original caffe github repository. I can search for the exact link and paste it here, but it is very easy to verify this either by trying to predict some images from training set using both the given model and [caffe demo](http://demo.caffe.berkeleyvision.org/) or searching through their github link

**Comment by abhijitbendale** [comment]
Check this link on [data preparation](http://caffe.berkeleyvision.org/gathered/examples/imagenet.html)
